tests/computation/data/fake_model_runner__old/supplier_model_runner.py

```diff
@@ -0,0 +1,33 @@
+from oasislmf.execution.runner import run as oasislmf_run
+
+
```

Comment:
Perhaps add some documentation of what this module is?

Reply:
Good point, added